style.css

```diff
@@ -360,14 +359,139 @@ span {
   height: 102px;
 }
 
+.contact-form {
+  background: #f9f9fa;
+  width: 375px;
+  height: 718px;
+}
+
+.contact-card {
+  padding: 1rem;
+}
+
+.contact-title {
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+}
+
+.contact-para {
+  width: 335px;
+  height: 63px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  color: #1a2236;
+  line-height: 21px;
+}
+
+.contact {
+  display: flex;
+  flex-direction: column;
+  gap: 0.3rem;
+}
+
+.contact label[for="name"],
+.contact label[for="email"],
+.contact label[for="message"] {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  color: #7d7c84;
+}
+
+.contact input[type="text"] {
+  width: 330px;
+  height: 52px;
+  background: #fff;
+  border-radius: 4px;
+}
+
+input[id="fname"] {
+  border: 1px solid #7d7c84;
+}
+
+input[id="mail"] {
+  border: 1px solid #7d7c84;
+}
+
+.form-btn {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions for active (when it's pressed) as shown in the Figma template below:

<img width="279" alt="image" src="https://user-images.githubusercontent.com/52934128/198187731-380b420a-fad9-4bfe-b901-788a7cb9f6b7.png">

I have provided a corrected version of your code below. Please review it and make the necessary change. If you have any questions or doubts, please let me know. The button on the pressed state has the `background: #FFFFFF;`, `color: #1A2236;`, `border-width: 2px` 

### EXAMPLE:
**for pressed state**
```
.form-btn:active {
   background-color:  #FFFFFF;;
   color: #1A2236
   border-width: 2px;
}
```